### PR TITLE
-

### PR DESCRIPTION
**Pull request by seanluse41**

### CODE REVIEW

File: servers/audio_server.h [hunk 1]
```diff
@@ -97,6 +97,7 @@ class AudioDriver {
 	virtual Error init() = 0;
 	virtual void start() = 0;
 	virtual int get_mix_rate() const = 0;
+	virtual int get_input_mix_rate() const { return get_mix_rate(); };
```
Comment (AThousandShips) [inline servers/audio_server.h:100]:
```suggestion
	virtual int get_input_mix_rate() const { return get_mix_rate(); }
```

File: drivers/coreaudio/audio_driver_coreaudio.mm [hunk 2]
```diff
@@ -405,13 +426,23 @@ Error AudioDriverCoreAudio::init_input_device() {
 			break;
 	}
 
-	mix_rate = _get_configured_mix_rate();
+#if MACOS_ENABLED
```
Comment (AThousandShips) [inline drivers/coreaudio/audio_driver_coreaudio.mm:429]:
```suggestion
#ifdef MACOS_ENABLED
```

File: drivers/coreaudio/audio_driver_coreaudio.mm [hunk 3]
```diff
@@ -116,7 +116,24 @@ Error AudioDriverCoreAudio::init() {
...
```
Comment (AThousandShips) [inline drivers/coreaudio/audio_driver_coreaudio.mm:119]:
```suggestion
#ifdef MACOS_ENABLED
```

**Comment by AThousandShips** [comment]
Any reason this isn't just a suggestion on the relevant PR?
* https://github.com/godotengine/godot/pull/88628

**Comment by seanluse41** [comment]
> Any reason this isn't just a suggestion on the relevant PR?
> 
> * [[macOS/iOS] Use hardware sampling rates for audio I/O. #88628](https://github.com/godotengine/godot/pull/88628)
> 
> Which was _already_ suggested here:
> 
> * [[macOS/iOS] Use hardware sampling rates for audio I/O. #88628 (comment)](https://github.com/godotengine/godot/pull/88628#discussion_r1527143516)
> 
> It really makes no sense to make a PR that changes the naming used in a different non-merged PR

Sorry, new to this, and the new gh cli thing.
Saw that the other PR had no action for two weeks on a fairly simple change, so I thought I'd give it a go.
I'll close it and try to figure out how to submit properly.

**Comment by AThousandShips** [comment]
That's not how things are done, just because something hasn't changed in a while doesn't mean it's abandoned, there's nothing for you to submit here, please just comment on that PR and explain these changes

**Pull request closed without merging**